xapian-letor/tests/Makefile.am

```diff
@@ -13,6 +13,7 @@ AM_CPPFLAGS = -I$(top_srcdir) -I$(top_srcdir)/common -I$(top_srcdir)/include \
 AM_CXXFLAGS += $(XAPIAN_CXXFLAGS)
 
 CLEANFILES =
+DISTCLEANFILES =
```

Comment:
There's no need to specify `DISTCLEANFILES` as empty here (I guess this is left over from when some files were being added to it, but they're now in `CLEANFILES` instead).

Reply:
fixed in a fixup commit.